libs/auth/src/drivers/interfaces/login-service.interface.ts

```diff
@@ -15,6 +15,8 @@ export interface DaffLoginServiceInterface<
 
   /**
    * Logs the user out by revoking their access token.
+   *
+   * @returns true if the token was revoked.
    */
-  logout(): Observable<void>;
```

Comment:
I would assume that this would throw an error into the stream if the token wasn't revoked. I'm not a big fan of this change.

Reply:
That is how I currently have it implemented in my WIP magento driver. It could also return `false`.

Reply:
I don't see any issue with returning void.

Reply:
Return `void` or throw an error if the revocation fails?

Reply:
What I'm saying is that there is no return type to this method. 

We log the user out. If it succeeds, great.

If we don't, we throw an error. A return value doesn't help in either of these situations.

Reply:
Is `and.returnValue(of())` the correct way to mock the successful `logout` behavior?

Reply:
You could test/mock this in many different ways. I think people would tend towards depending on the testing service, but if you wanted to mock it, yes, you could mock it that way.